mmv1/third_party/terraform/services/accesscontextmanager/resource_access_context_manager_service_perimeter_egress_policy_test.go

```diff
@@ -30,11 +30,6 @@ func testAccAccessContextManagerServicePerimeterEgressPolicy_basicTest(t *testin
 			{
 				Config: testAccAccessContextManagerServicePerimeterEgressPolicy_basic(org, policyTitle, perimeterTitle),
 			},
-			{
-				ResourceName:      "google_access_context_manager_service_perimeter.test-access",
```

Comment:
Why are we removing import tests from these other resources?

Reply:
They are the non dry-run version of the same resource. They should not have imports either so I am removing them to make it consistent.